common/dev/isl69260.c

```diff
@@ -0,0 +1,76 @@
+#include "isl69260.h"
+
+#include <stdio.h>
+
+#include "sensor.h"
+#include "hal_i2c.h"
+#include "plat_sensor_table.h"
+
+#define RETRY 5
+
+struct isl69260_page_data isl69260_page_data_args[] = { [0] = { 0x0, 0x0 }, [1] = { 0x0, 0x1 } };
+
+int isl69260_switch_page(uint8_t sensor_num, void *args)
```

Comment:
check args != NULL

Reply:
The optimization code is based on v2022.07.01. After that, the main branch has been phased in some
commits related to sensor monitoring. Those changes will be phased in main_v2 in the other PRs. This
problem will be solved in those PRs.
Please check #223  .

Reply:
The optimization code is based on v2022.07.01. After that, the main branch has been phased in some
commits related to sensor monitoring. Those changes will be phased in main_v2 in the other PRs. This
problem will be solved in those PRs.
Please check #223  .